app.rb

```diff
@@ -0,0 +1,79 @@
+require_relative 'person'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'nameable'
+require_relative 'trimmer_decorator'
+require_relative 'capitalize_decorator'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_all_books
+    puts 'List of all books:'
+    @books.each do |book|
+      puts "#{book.title} by #{book.author}"
+    end
+  end
+
+  def list_all_people
+    puts 'List of all people:'
+    @people.each do |person|
+      puts "#{person.id}, #{person.name}, #{person.age}"
+    end
```

Comment:
- [ ] When displaying the peoples list you should display their indexes to be able to use them while creating a rental, and ID's to be able to see a list of rentals to the specific person

Reply:
resolved thanks